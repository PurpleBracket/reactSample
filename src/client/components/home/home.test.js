/* global describe test expect */
import Home from './home'
import React from 'react'
import { shallow } from 'enzyme'

describe('Home', () => {
  test('Snapshot', () => {
    const home = shallow(<Home />)
    expect(home).toMatchSnapshot()
  })
})
