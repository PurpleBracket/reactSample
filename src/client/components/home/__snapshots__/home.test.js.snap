// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot 1`] = `
<div
  className="homeContainer"
>
  <div
    className="container"
  >
    <div
      className="columns"
    >
      <div
        className="column is-three-fifths is-offset-one-fifth"
      >
        <br />
        <br />
        <br />
        <br />
        <br />
        <Headline
          color="white"
          size="1"
          spaced={true}
        >
          Welcome to Treasure Landing Page!
        </Headline>
        <Subhead
          size="3"
        >
          Start growing with Treasure
        </Subhead>
        <Button
          color="white"
          style="outline"
        >
          Learn More
        </Button>
      </div>
    </div>
  </div>
</div>
`;
